src/clusterfuzz/_internal/bot/tasks/utasks/progression_task.py

```diff
@@ -111,6 +112,22 @@ def handle_progression_revision_list_error(
                                          'Failed to fetch revision list')
 
 
+def _cleanup_stacktrace_blob_from_storage(
+    uworker_output: uworker_msg_pb2.Output):
+  """Cleanup the blob created in preprocess if it wasn't used to store the
+  filterd stacktrace."""
+  if (uworker_output.HasField('progression_task_output') and
+      uworker_output.progression_task_output.last_tested_crash_stacktrace.
```

Comment:
This is very hard to read. Make this a second condition afterwards or something please.

Reply:
Done.